addons/pos_self_order/static/src/mobile/pages/product_main_view/product_main_view.xml

```diff
@@ -78,15 +78,31 @@
                                                 t-model="state.selectedVariants[attribute.id]"
                                                 />
                                         </label>
-                                        <span class="flex-grow-1">
+                                        <span class="px-2 py-1 ms-2 rounded-pill text-bg-info fs-6">
                                             <t t-esc="value.name"/>
-                                            <t t-if="value.price_extra.list_price">
-                                                + <t t-esc="selfOrder.formatMonetary(value.price_extra.list_price)" />
+                                            <t t-if="value.price_extra.display_price">
+                                                + <t t-esc="selfOrder.formatMonetary(value.price_extra.display_price)" />
                                             </t>
                                         </span>
                                     </li>
                                 </ul>
                             </div>
+                            <div t-elif="attribute.display_type === 'multi'">
```

Comment:
this code should really be shared between the `pos`, the `self` and the `kiosk`. We can do this after this pr, though